nevergrad/functions/helpers.py

```diff
@@ -109,7 +110,8 @@ def create_crossvalidation_experiments(
 
         """
         funcs: tp.List["SpecialEvaluationExperiment"] = []
-
+        if "PYTEST_NEVERGRAD" in os.environ:
+            pareto_subset_methods = ("random",)  # override for speed
```

Comment:
this is a *slightly* cleaner way of reducing the test, but that should not become a habit, because having such cases in production code will end up creating weird problems (silencing stuff that should have gone wrong)
so never, ever in the outside of `nevergrad/functions`, at the very least